service/controller/v13/resource/cloudformation/spec.go

```diff
@@ -10,6 +10,10 @@ const (
 	workerRoleKey = "WorkerRole"
 
 	namedIAMCapability = "CAPABILITY_NAMED_IAM"
+
+	// versionBundleVersionParameterKey is the key name of the Cloud Formation
+	// parameter that sets the version bundle version.
+	versionBundleVersionParameterKey = "VersionBundleVersionParameter"
```

Comment:
Is it a common practice to add `Parameter` suffix to a CF parameter name?

Reply:
Yes the AWS docs all have the Parameter suffix. I think it's because the Ref function is used and it makes it clear what is being referenced in the template.

https://github.com/giantswarm/aws-operator/blob/505a55b2000960134b02d68d8b30efa6b285d678/service/controller/v13/templates/cloudformation/guest/outputs.go#L23-L25

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html

Reply:
Makes sense 👍